app/forms/security_event_form.rb

```diff
@@ -213,7 +213,22 @@ def identity
     return if event.blank? || !service_provider
     return @identity if defined?(@identity)
 
-    @identity = Identity.find_by(
+    @identity = if service_provider.agency_id
+                  identity_from_agency_identity
```

Comment:
You might want to default to identity if agency_identity comes up short so make this an or instead of an if

Reply:
But if `agency_id` is nil, it'll mess up the query on `agency_identities` table, right?

Reply:
shouldn't mess up the query but not really a requirement to change this because in prod we should never see a missing agency_identity and this feature will probably not be used in int so we might be good.

Reply:
ok cool thanks, I'll leave it for now